nbs/raster_calculation_blocks.py

```diff
@@ -0,0 +1,2 @@
+# %%
```

Comment:
file is leeg = verwijderen?

Reply:
nee hier moet nog een voorbeeld inkomen van hoe block berekeningen gaan.